robottelo/entities.py

```diff
@@ -316,8 +337,7 @@ def create_missing(self):
         # generate an URL pointing to a docker server
         if provider.lower() == 'docker':
             if 'url' not in vars(self):
-                self.url = random.choice((
-                    get_internal_docker_url(), get_external_docker_url()))
+                self.url = self._server_config.url
```

Comment:
Shoot, you're right.

NailGun has no way of knowing about the settings in Robottelo's config file, such as the location of a docker compute resource. As a result, I've opted to make NailGun simply assume that a docker compute resource is available on the server. It's an assumption, but it's the best assumption available given the constraints on what knowledge NailGun has. I'll fix this to assign an address such as the following:

    self.url = '{0}:2325'.format(self._server_config.url)